src/renderer/pages/Contact/ContactAll.vue

```diff
@@ -1,27 +1,46 @@
 <template>
-  <div class="ContactAll relative bg-theme-feature rounded-lg m-r-4 p-10">
-    <div class="block h-full">
-      <div class="ContactAll__header">
-        <h3>{{ $t('PAGES.CONTACT_ALL.HEADER') }}</h3>
-
-        <div class="flex items-center">
-          <button
-            v-if="!hasWalletGridLayout"
-            class="ContactAll__CreateButton"
-            @click="createContact"
-          >
-            <span class="ContactAll__CreateButton__icon">
-              <SvgIcon
-                name="plus"
-                view-box="0 0 9 9"
-                class="text-center"
-              />
-            </span>
-
-            <span class="flex items-center h-10 px-4">
-              {{ $t('PAGES.CONTACT_ALL.CREATE_CONTACT') }}
-            </span>
-          </button>
+  <div class="ContactAll flex flex-col">
+    <div class="ContactAll__header flex flex-row space-between bg-theme-feature rounded-lg px-10 py-2 mb-3">
```

Comment:
Can we change this to py-5 instead of py-2, I think the identicon looks a little close to the edge:

py-2:
![image](https://user-images.githubusercontent.com/8069294/51679857-f7f57880-1fd7-11e9-8e42-95702b51e3cf.png)

py-5:
![image](https://user-images.githubusercontent.com/8069294/51679830-e9a75c80-1fd7-11e9-8709-fd77ee91c32c.png)

Reply:
I've changed it. I was feeling the same way, but I tried to follow the original design.

Reply:
@olejegcord could you confirm this change?

Reply:
Please make "I" more bold 2-3 times.
"py-5" - ok =)